app/Filament/Resources/ServerResource.php

```diff
@@ -24,22 +32,94 @@
 use Filament\Tables\Table;
 use Illuminate\Database\Eloquent\Builder;
 use Illuminate\Database\Eloquent\SoftDeletingScope;
-use phpDocumentor\Reflection\Types\Self_;
+
 
 class ServerResource extends Resource
 {
     public const ICON = 'tabler-server-2';
 
     protected static ?string $model = Server::class;
-
     protected static ?string $navigationIcon = self::ICON;
+    protected static ?string $navigationLabel = 'Runners';
 
+    protected static ?string $modelLabel = 'Runners';
```

Comment:
i would rather rename the class(es) and make the necessary migrations than have a class that is named so differently from the label.